scrabble.js

```diff
@@ -1,14 +1,129 @@
+const scrabbleLetters = {
+     a: 1, b: 3, c: 3, d: 2, e: 1, f: 4,
+     g: 2, h: 4, i: 1, j: 8, k: 5, l: 1,
+     m: 3, n: 1, o: 1, p: 3, q: 10, r: 1,
+     s: 1, t: 1, u: 1, v: 4, w: 4, x: 8,
+     y: 4, z: 10,
+   };
+
 const Scrabble = {
-  score: function(word) {
-    // TODO: implement score
-  }
+ score: function(word) {
+
+   let regex = /^[a-zA-Z]+$/;
+
+   if (!regex.test(word)) {
+     throw new Error('Invalid characters');
+   }
+
+   if (word.length > 7) {
+     throw new Error('Word must be 1 - 7 characters long');
+   }
+
+   let lowercaseWord = word.toLowerCase();
+   let score = 0;
+
+
+   for (let i = 0; i < lowercaseWord.length; i++) {
```

Comment:
This works fine, you can also use [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce?v=a) to calculate it.